ignite/cmd/plugin.go

```diff
@@ -62,6 +78,23 @@ func parseLocalPlugins(cmd *cobra.Command) (*pluginsconfig.Config, error) {
 	return pluginsconfig.ParseDir(wd)
 }
 
+func parseGlobalPlugins() (cfg *pluginsconfig.Config, err error) {
+	globalDir, err := plugin.PluginsPath()
+	if err != nil {
+		return cfg, err
+	}
+
+	cfg, err = pluginsconfig.ParseDir(globalDir)
+	if err != nil {
+		return &pluginsconfig.Config{}, nil
+	}
```

Comment:
Why is the error not returned? I'm wondering if there is a reason. If so maybe it would be good to leave a comment.

Reply:
If we cannot parse for the global config, we just don't load it and continue execution.  Perhaps it makes sense to return a typed error?

Reply:
> If we cannot parse for the global config, we just don't load it and continue execution. Perhaps it makes sense to return a typed error?

I believe a comment should be enough then :), to make it clear to others that this snippet should not be changed to return the error.